webrtc-c/canary/src/CanarySignaling.cpp

```diff
@@ -305,7 +305,7 @@ STATUS run(Canary::PConfig pConfig)
     // Generate a random channel name if not specified in the config.
     // In case we generate the random name we will follow-up with deleting
     // it upon exit to prevent the account from ever increasing channel count
-    if (pConfig->channelName.value == NULL || IS_EMPTY_STRING(pConfig->channelName.value)) {
+    if (pConfig->channelName.value[0] == '\0' || IS_EMPTY_STRING(pConfig->channelName.value)) {
```

Comment:
Actually, my bad - I forgot that this is an array rather than a pointer. We should just remove the first check and only leave the IS_EMPTY_STRING one

Reply:
Ouch, sorry, I just realized that..

Reply:
Updated.